src/tests/topp/THIRDPARTY/PercolatorAdapter_1_out.idXML

```diff
@@ -40,7 +40,7 @@
 				<UserParam type="int" name="Percolator:klammer" value="0"/>
 				<UserParam type="string" name="Percolator:fasta" value=""/>
 				<UserParam type="string" name="Percolator:decoy_pattern" value="random"/>
-				<UserParam type="string" name="Percolator:post_processing_tdc" value="false"/>
```

Comment:
but why is here the string parameter?

Reply:
that's the INI part of the story... somehow, the other twin of the duplicated parameters wins here...
But I'm not sure it also dominates when doing 'getStringOption_()' in TOPP's main_()